apps/ckit-app/src/components/Wallet/WalletConnect.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from 'antd';
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { WalletContainer, DisplayWalletName } from 'containers/WalletContainer';
+
+export const WalletConnectFC: React.FC = () => {
+  const { currentWalletIndex, wallets, setModalVisible } = WalletContainer.useContainer();
```

Comment:
same with above, can be refactor with `selectedWallet`

Reply:
fixed